gatsby-node.js

```diff
@@ -3,6 +3,16 @@ const { newCloudinary, getResourceOptions } = require('./utils');
 const REPORTER_PREFIX = `gatsby-source-cloudinary`;
 const NODE_TYPE = `CloudinaryMedia`;
 
+exports.createSchemaCustomization = ({ actions }) => {
+  const { createType } = actions;
+
+  createType(`
+    type CloudinaryMedia implements Node {
+      joinedAt: Date
```

Comment:
This will be good to add after I add support for gatsby-transformer-cloudinary, but the type fields should then be: publicId, cloudName, originalHeight, originalWidth, originalFormat.